themes/powerline/powerline.theme.bash

```diff
@@ -2,6 +2,8 @@
 
 . "$BASH_IT/themes/powerline/powerline.base.bash"
 
+CAN_SUDO=true
```

Comment:
After re-reading your changes, I can see where the confusion comes from. You asked users to change the value in the theme file. That's not the way we want to do customizations. This is how to do it:

* The theme file sets the variable and defines a default value if the variable does not exist yet. This is what the `${THEME_CHECK_SUDO:=true}` does. It checks for the existence of the variable, and uses the variables value if the variable is already defined. If it not defined, it sets it to `true` as a default value.
* If the user wants to override the default value, they can make that change in their Bash configuration file (`~/.bash_profile` or `~/.bashrc`) and don't have to change the theme file.